tethysapp/hydro_var_monitor/controllers.py

```diff
@@ -1,49 +1,125 @@
-from django.http.response import JsonResponse
+import json
+import logging
+import os
 from datetime import date
 from datetime import datetime
-from django.shortcuts import render
-from .ee_auth import *
-import json
 from django.http import JsonResponse, HttpResponseNotAllowed
+from django.http.response import JsonResponse
+from django.shortcuts import render
+from django.urls import path
+# from . import views
 from tethys_sdk.routing import controller
+
 from . import ee_auth
-import logging
-import os
+from .compare import air_temp_compare, precip_compare
+from .ee_auth import *
 from .ee_tools import ERA5, get_tile_url, GLDAS, CHIRPS, IMERG, GLDAS_evapo
 from .plots import plot_ERA5, plot_GLDAS, plot_IMERG, plot_CHIRPS
-from .compare import air_temp_compare, precip_compare
+from .unzip import unzip_and_read_files
+import zipfile
+import tempfile
 
+from .app import HydroVarMonitor as App
 
-@controller(name='home', url='hydro-var-monitor')
-def home(request):
-    if not EE_IS_AUTHORIZED:
-        return render(request, 'hydro_var_monitor/no_auth_error.html')
+
+@controller(name='home', url='hydro-var-monitor', app_workspace=True)
+def home(request, app_workspace):
+    lang = request.GET.get('lang', 'en')
+    if lang not in ['en', 'es']:
+        lang = 'en'
+
+    # if not EE_IS_AUTHORIZED:
+    #     return render(request, 'hydro_var_monitor/no_auth_error.html')
```

Comment:
this block of code should not be commented.

Reply:
fixed